ansible-st2-local/roles/arteria/tasks/main.yml

```diff
@@ -1,35 +0,0 @@
----
-
-- include: arteria_mailclient.yml
-  tags: arteria_mailclient
-
-- include: conf.yml
-  tags: arteria_master_conf
-
-- name: Pause for 1 minute until stackstorm services are up and online
-  pause: minutes=1
-
-- include: register_components.yml
-  tags: arteria_register_components
-
-- name: Override paramiko setting (currently buggy)
```

Comment:
We should add a issue in the backlog about this so we don't forget to change this back when we upgrade st2 in the future.